openedx/core/lib/xblock_utils.py

```diff
@@ -211,6 +211,23 @@ def grade_histogram(module_id):
     return grades
 
 
+def add_grading_markup(course, block, view, frag, context):  # pylint: disable=unused-argument
```

Comment:
@den99f see function around. Docstring format is incorrect. Please update is so:

"""
Updates the supplied module with a new get_html function that wraps the output of the old get_html function with additional grading information.
"""